specificatie/BRK-Bevragen/APILAB2603/openapi.yaml

```diff
@@ -939,71 +929,54 @@ components:
       enum:
       - "ingeschreven_niet_natuurlijk_persoon"
       - "kadaster_niet_natuurlijk_persoon"
-    Objectlocatie:
-      allOf:
-      - $ref: "#/components/schemas/Adreslocatie"
-      - description: "Een Objectlocatie is een ObjectlocatieBinnenland of een ObjectlocatieBuitenland."
-    Objectlocatiebuitenland:
-      allOf:
-      - $ref: "#/components/schemas/Objectlocatie"
-      - description: "Een objectlocatiebuitenland is een adres buiten Nederland."
-        properties:
-          adres:
-            type: "string"
-            title: "adres"
-            description: "Het adres is een combinatie van de straat en huisnummer."
-          regio:
-            type: "string"
-            title: "regio"
-            description: ""
-          woonplaats:
-            type: "string"
-            title: "woonplaats"
-            description: "Woonplaats is De postcode/woonplaats is de combinatie van\
-              \ een eventuele postcode en woonplaats."
-          land:
-            allOf:
-            - $ref: "#/components/schemas/Waardelijst"
-            - description: "Mogelijke waarden zijn te vinden in deze [Waardelijst](http://www.kadaster.nl/schemas/waardelijsten/BRPLand/)"
-    ObjectlocatieBinnenland:
+    Adres:
+      properties:
+        huisnummer:
+          type: "integer"
+          description: "URI https://bag.basisregistraties.overheid.nl/doc/begrip/Huisnummer"
+        huisletter:
+          type: "string"
+          description: "URI https://bag.basisregistraties.overheid.nl/doc/begrip/Huisletter"
+          example: "A"
+        huisnummerToevoeging:
+          type: "string"
+          description: "URI https://bag.basisregistraties.overheid.nl/doc/begrip/Huisnummertoevoeging"
+          example: "3"
+        straat:
+          type: "string"
+          example: "Baron Schimmelpenninck van der Oyelaan"
+        postcode:
+          type: "string"
+          example: "1234AA"
+        woonplaats:
+          type: "string"
+          example: "Nootdorp"
+        adresBuitenland:
```

Comment:
Als buitenlands en binnenlands adres wordt plat geslagen, is het dan niet beter om dan internationaal adressering te gebruiken?  straat en adresBuitenland zijn dan adresseringregel1 & adresseringregel2. WoonplaatsBuitenland is dan ook niet nodig

Reply:
ik heb het nu gewijzigd naar adresregel1, adresregel2 en adresregel3. De eerste twee worden altijd gevuld, zowel bij een Nederlands adres, een buitenlands adres en een postbus.
Adresregel1 is straat + huisnummer + huisletter + huisnummertoevoeging of "postbus" + postbusnummer of adres buitenland
Adresregel2 is postcode + woonplaats of woonplaats buitenland
Adresregel3 is de regio van een buitenlands adres

Op deze manier kan een gebruiker kiezen de lossen velden van een Nederlands adres te gebruiken (zoals postcode en huisnummer) of samengestelde adresregels. 
Een gebruiker die alleen het volledige adres wil, hoeft nu geen onderscheid meer te maken tussen de verschillende soorten adressen.